timber-sample/src/main/java/com/example/timber/ExampleApp.java

```diff
@@ -15,6 +15,10 @@
     } else {
       Timber.plant(new CrashReportingTree());
     }
+
+    int size = 10;
+    int i = 5;
+    Timber.i("Position %d", size - i);
```

Comment:
Oops, didn't mean to commit that. But yeah - on master:

```
ExampleApp.java:21: Warning: Replace String concatenation with Timber's string formatting [BinaryOperationInTimber]
    Timber.i("Position %d", size - i);
                            ~~~~~~~~

   Explanation for issues of type "BinaryOperationInTimber":
   Since Timber handles String#format() automatically, use this instead of
   String concatenation.
```
with this change, it's gone. I'll revert that change to `ExampleApp.java` though.